pkgs/development/tools/sublime-merge/common.nix

```diff
@@ -0,0 +1,87 @@
+{buildVersion, x64sha256}:
+
+{ fetchurl, stdenv, glib, xorg, cairo, gtk2, pango, makeWrapper, openssl, bzip2,
+  pkexecPath ? "/run/wrappers/bin/pkexec", libredirect,
+  gksuSupport ? false, gksu, unzip, zip, bash}:
+
+assert gksuSupport -> gksu != null;
+
+let
+
+  libPath = stdenv.lib.makeLibraryPath [glib xorg.libX11 gtk2 cairo pango];
+  redirects = [ "/usr/bin/pkexec=${pkexecPath}" ]
+    ++ stdenv.lib.optional gksuSupport "/usr/bin/gksudo=${gksu}/bin/gksudo";
+in let
+
+  # package with just the binaries
+  sublimeMerge = stdenv.mkDerivation {
+    name = "sublimeMerge-${buildVersion}-bin";
+    src =
+      fetchurl {
+        name = "sublime_merge_${buildVersion}_x64.tar.xz";
+        url = "https://download.sublimetext.com/sublime_merge_build_${buildVersion}_x64.tar.xz";
+        sha256 = x64sha256;
+      };
+
+    dontStrip = true;
+    dontPatchELF = true;
+    buildInputs = [ makeWrapper zip unzip ];
+
+    buildPhase = ''
+      for i in crash_reporter git-credential-sublime ssh-askpass-sublime  sublime_merge; do
+        patchelf \
+          --interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+          --set-rpath ${libPath}:${stdenv.cc.cc.lib}/lib${stdenv.lib.optionalString stdenv.is64bit "64"} \
+          $i
+      done
+    '';
+
+    installPhase = ''
+      # Correct sublime_merge.desktop to exec `sublime' instead of /opt/sublime_merge
+      sed -e "s,/opt/sublime_merge/sublime_merge,$out/sublime_merge," -i sublime_merge.desktop
+
+      mkdir -p $out
+      cp -prvd * $out/
+
+      # We can't just call /usr/bin/env bash because a relocation error occurs
+      # when trying to run a build from within Sublime Merge
+      ln -s ${bash}/bin/bash $out/sublime_bash
+      wrapProgram $out/sublime_bash \
+        --set LD_PRELOAD "${stdenv.cc.cc.lib}/lib${stdenv.lib.optionalString stdenv.is64bit "64"}/libgcc_s.so.1"
```

Comment:
This is weird so. Does it not like `libredirect`? You could use `makeWrapper` to create `sublime_bash` directly instead of having the symlink first.